templates/zerver/features.html

```diff
@@ -306,6 +306,31 @@
                     </div>
                 </div>
             </div>
+            <div class="feature-block left">
+                <div class="flex">
+                    <div class="icon-container">
+                        <i class="fa-line-chart icon-vector-3x feature-icon"></i>
+                    </div>
+                    <div class="block">
+                        <h4>{% trans %}Analytics{% endtrans %}</h4>
+                        <p>{% trans %}View detailed graphs and statistics of your organization's activity.
+                            {% endtrans %}</p>
+                    </div>
+                </div>
+            </div>
+            <div class="feature-block">
+                <div class="flex">
+                    <div class="icon-container">
+                        <i class="fa-language icon-vector-3x feature-icon"></i>
+                    </div>
+                    <div class="block">
+                        <h4>{% trans %}Fully internationalized{% endtrans %}</h4>
+                        <p>{% trans %}Zulip has full support for Unicode, and the core Zulip UI has
```

Comment:
Yeah; we probably need to add it to the exclude list in `tools/lib/capitalization.py`.